doc/en/user/source/rest/api/services.rst

```diff
@@ -215,4 +215,74 @@ Controls Web Map Service settings for a given workspace.
      -
      -
 
+``/services/wmts/settings[.<format>]``
```

Comment:
I am not sure this page needs to be maintained, idea was to replace it with the Open API documentation. We just have not removed these as we are not 100% sure we captured everything.

Reply:
well in any case it's now just as useless as the other ones